schema/Mutation.graphql

```diff
@@ -84,4 +84,9 @@ type Mutation {
     Revoke user session
     """
     revokeUserSession(id: ObjectID!, displayNotice: Boolean): Boolean!
+
+    """
+    Create new lives
+    """
+    createLife(firstName: String!, lastName: String!, description: String!, birthday: DateTime!, hobbies: [String]!): Life!
```

Comment:
Will be better if you wrap this in an input object instead for your arguments.
`firstName: String!, lastName: String!, description: String!, birthday: DateTime!, hobbies: [String]!` 

Also, suggest that your hobbies type is `[String!]!` 

`[String]!` means you can have null values in your array but we don't want that